context/app/static/js/hooks/useEntityData.ts

```diff
@@ -2,12 +2,30 @@ import { useMemo } from 'react';
 import { useSearchHits } from 'js/hooks/useSearchData';
 import { Entity } from 'js/components/types';
 
+export const useEntityQuery = (uuid: string | string[], source?: string[]) => {
+  return useMemo(
+    () => ({
+      query: { ids: { values: typeof uuid === 'string' ? [uuid] : uuid } },
+      _source: source,
+    }),
+    [uuid, source],
+  );
+};
+
 export function useEntityData(uuid: string, source?: string[]): [Entity, boolean] {
-  const query = useMemo(() => ({ query: { ids: { values: [uuid] } }, _source: source }), [uuid, source]);
+  const query = useEntityQuery(uuid, source);
 
   const { searchHits, isLoading } = useSearchHits<Entity>(query);
 
   return [searchHits[0]?._source, isLoading];
 }
 
+export function useEntitiesData(uuids: string[], source?: string[]): [Entity[], boolean] {
+  const query = useEntityQuery(uuids, source);
+
+  const { searchHits, isLoading } = useSearchHits<Entity>(query);
+
+  return [searchHits.map((hit) => hit._source), isLoading];
```

Comment:
Slightly prefer an obj so we can also include the other returned values from `useSearchHits`.

Reply:
>other returned values

From what I can tell, `useSearchHits` only returns the `searchHits` and `isLoading` values, so I stuck with the same format that `useEntityData` uses above. Are you interested in us keeping the top-level `hit` data?